src/components/CustomExtendedDataField/CustomExtendedDataField.js

```diff
@@ -117,7 +134,7 @@ const CustomFieldLong = props => {
       }}
       label={label}
       placeholder={placeholder}
-      {...validateMaybe}
+      validate={value => validate(value, maximum, minimum)}
```

Comment:
Check my other comment.
`validate={validators.validateInterger(max, min, numberTooSmallMessage, numberTooBigMessage)}`

Reply:
perhaps the order: `minimum, maximum` is more clean.

Reply:
Should the validate function also be in the utils file? I'm referring to this one

```
   const validate = (value, min, max) => {
      const requiredError = isRequired && value != null ? undefined : (requiredMessage || defaultRequiredMessage);
      const numberError = validateInteger(value, max, min, numberTooSmallMessage, numberTooBigMessage);
      return requiredError || numberError; 
    };
```

 I feel like it could also be left here as this specific validation logic is only relevant to this single component.

Reply:
The util/validators.js is the default place where customizers should try to find validators for React Final Form Fields.

Other validation needs can be more ad hoc.